.github/workflows/ci.yml

```diff
@@ -13,11 +13,7 @@ jobs:
         os: [ubuntu-22.04, macos-12]
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: DeterminateSystems/nix-installer-action@main
-        with:
-          # Allow the installed Nix to make authenticated Github requests.
-          # If you skip this, you will likely get rate limited.
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: EricCrosson/nix-installer-action@3-add-default-value-for-the-the-github-token-input
```

Comment:
Mind pointing this back to the detsys repoo and we can merge it? :)

Reply:
Sure! I made a new PR instead of cleaning up all the metadata on this one: https://github.com/DeterminateSystems/nix-installer-example/pull/6